presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/TestIcebergSmoke.java

```diff
@@ -84,9 +95,14 @@ public void testShowCreateSchema()
                         "\\)");
     }
 
-    @Test
     @Override
     public void testDescribeTable()
+    {
+        testDescribeTableForFormat(ORC);
+    }
+
+    @Test(dataProvider = "allFormats")
+    public void testDescribeTableForFormat(FileFormat format)
```

Comment:
rename to `testDescribeTable` and drop method above?

Reply:
Both methods are needed because of `AbstractTestIntegrationSmokeTest`, as in the previous comment.